server/routes/alumni.js

```diff
@@ -79,22 +79,27 @@ router.get('/all/:timezone', async (req, res, next) => {
         })
         res.json({'alumni' : alumni});
     } catch (e) {
-        console.log("Error: util#allAlumni", e);
+        console.log("Error: alumni#allAlumni", e);
         res.status(500).send({'error' : e});
     }
 });
 
-router.get('/:id', async (req, res, next) => {
+router.get('/one/:id', async (req, res, next) => {
```

Comment:
Put `/one` back in since this was the first part of any `GET` route was getting parsed as `id`!

Reply:
👍